src/Lecture1.hs

```diff
@@ -119,4 +141,17 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String 
+lowerAndGreater n list = (show n) ++ " is greater than " ++ (show (findGreater n list)) ++ " elements and lower than " ++ (show (findLower n list)) ++ " elements"
+    where 
+        findLower :: Int -> [Int] -> Int
+        findLower _ [] = 0
+        findLower n (x:xs)
+            | x > n     = 1 + findLower n xs
+            | otherwise = findLower n xs
```

Comment:
You don't change `n` during recursive calls. In that case, you don't need to pass it explicitly as `n` is already in scope (functions inside `where` can see arguments of the main function). This will allow you to simplify this function a bit.